config/initializers/assets.rb

```diff
@@ -11,5 +11,6 @@
 # Rails.application.config.assets.precompile += %w( adp_swagger_application.js *.png)
 # Rails.application.config.assets.precompile += %w( application.test.js ) if Rails.env.test?
 # Rails.application.config.assets.precompile += %w( pdf.css )
-Rails.application.config.assets.paths << Rails.root.join('node_modules', 'govuk-frontend', 'govuk', 'assets', 'images')
+Rails.application.config.assets.paths << Rails.root.join('node_modules', 'govuk-frontend', 'dist', 'govuk', 'assets')
+Rails.application.config.assets.paths << Rails.root.join('node_modules', 'govuk-frontend', 'dist', 'govuk', 'assets', 'images')
```

Comment:
You asked about the possibility of removing this line. Testing locally, the app breaks without it. There may be a way to configure it so that adding the `/assets` directory is sufficient to include the `/images` subdirectory but I haven't been able to work out what it is and I have no objection to leaving this as it is.